crates/bevy_transform/src/components/transform.rs

```diff
@@ -277,3 +277,9 @@ impl Mul<Vec3> for Transform {
         self.mul_vec3(value)
     }
 }
+
+impl From<Vec3> for Transform {
```

Comment:
Docs plz. While "into translation" is the obvious conversion, it's not the only possible one.